docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3"
+
+services: 
+     
+  db:
+    image: mariadb:latest
+    container_name: mariadb-server
+    restart: always
+    command: --default-authentication-plugin=mysql_native_password
+    environment:
+      MYSQL_ROOT_PASSWORD: reidb  
+      #MYSQL_USER: reibegaj
+      #MYSQL_PASSWORD: reidatabase
+    volumes:
+      - mariadb:/var/lib/mysql
+
+  
+  web:
+    image: phpmyadmin:latest
+    container_name: phpmyadmin
+    links:
+      - "db:mariadb-server"
+    ports:
+      - "80:80"
+    environment:
+     MYSQL_ROOT_PASSWORD: reidb
+     PMA_HOST: mariadb-server
+     PMA_USER: root
+     PMA_PASSWORD: reidb
+
+  app:
+    build:
```

Comment:
Do not store the app in the root directory. Build it from its own separate directory.